pom.xml

```diff
@@ -57,7 +57,7 @@
         <killbill-api.version>0.50.2</killbill-api.version>
 
         <osgi.version>5.0.0</osgi.version>
-        <version.checkstyle>2.16</version.checkstyle>
+        <version.checkstyle>3.1.0</version.checkstyle>
```

Comment:
Do we have breaking changes?

Reply:
Nothing breaks in killbill-authnet-plugin and killbill-resources. Testing with billing-service and swiper-service will take some time

Reply:
Also, I think this is a checkstyle plugin and _probably_ only used at build time. I don't think it should affect anything else.